src/scala/com/github/johnynek/bazel_deps/Normalizer.scala

```diff
@@ -55,14 +55,22 @@ object Normalizer {
           table(n).collect { case (_, Right(v)) => v }.toSet.size > 1
       }
 
-      def disambiguate(node: Node): Table =
+      def disambiguate(node: Node, visited: List[Node] = List()): Table = {
```

Comment:
can you:

1. not use the default arg here? It is dangerous since if you forget to put it in a recursion you wipe out the visited set.
2. can we use a `Set[Node]` instead, which gives O(1) addition and membership check? `List[Node]` has O(1) addition, but `O(N)` membership check.